### ISSUE TITLE
I have a new router and it changed the IP address; how do I reconnect with Lennox S30

### ISSUE THREAD

**Issue by kkolkebeck** [PeteRager/lennoxs30#352]
I have been running your Lennox Integration for several years with few problems. I had a router failure and had to make a change and the new router changed the basic IP address structure from 192.168.1.## to 192.167.50.##. As a result I lost the connection to the Thermostat, even though it is still visible on the router. I tried to find a way to change to the new address but was unsuccessful. I tried updating your integration but it failed. Same with HACS. So I removed both and reinstalled with newest versions. 
Both now appear. HACS is running fine. But while Lennox says it is, it does not show up in the list of integrations like it did. While it allows me to past to the dashboard, it does not link to any data. Also, there is no Lennox Log. 
How do I need to correct this? Should I delete the integration and start all over?

**Comment by kkolkebeck** [comment]
I figured it all out.

**Comment by curtgrimes** [comment]
Can you share what you did to figure it out? It may [help others](https://xkcd.com/979/) that come across this issue and experience the same symptoms as you.

**Comment by kkolkebeck** [comment]
Pete,

Evidently my problems started because each of my instances, HACS and Lennox had not been updated and so they did not have the needed directories. So….. 
I deleted each, first Lennox, then HACS. 
Then I deleted the associated configuration files and folders, although the top level folders would not delete even though empty. 
Then I changed the folder names; oldLennox, oldHACS. 
Then I deleted the template sensors I had created. (Actually I just copied them to a text file.)
Then I did an install on the HACS integration. 
Then I did an install on your Lennox integration. 
Then I did a configure and setup on your integration.
Then I put the template sensors back.

I did a reboot in between steps, sometimes two where suggested as with HACS. All came back. There must have been some remnants from the first install as my climate-dot name for the variables was the same as before, but perhaps that is just because it gets it from the “Name” in the General tab of the System setup. I did not have to change any variable names and all my displays came back up on line. 

Thank you for the help and the product. 

Ken Kolkebeck
